.github/workflows/git.yml

```diff
@@ -1,6 +1,6 @@
 name: Git Checks
 
-on: [push]
+on: [pull_request]
```

Comment:
According to the documentation and "documentation": https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request
> Note: By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.

https://github.community/t/what-is-a-pull-request-synchronize-event/14784/2
> What this implies is that if you push a new commit to the HEAD ref of a pull request then this “synchronize” event will be triggered. This is because the system is syncing the pull requests with the latest changes. This will NOT trigger if the base ref is updated.

Ah, it is effectively the same for our purposes, but Github's documentation is not clear.

Reply:
Right, I actually noticed this before, it's not new (there's also no `push` events when changing the base of a PR). I guess one would have to add `types: [assigned, opened, synchronize, reopened, edited]` to correctly handle that, although that will also trigger if just the description or title is updated. Should I add that?

Reply:
🤔 This _technically_ doesn't need to run on a base ref change, since it only matters if the commits themselves change.